### ISSUE TITLE
AssemblyScript Public Meeting #25 - November 18th, 2020

### ISSUE THREAD

**Issue by torch2424** [AssemblyScript/working-group#50]
# Date and Time

This public meeting will take place: **November 18th, 2020, 18:00 UTC (10:00 AM US PDT, UTC -8)**

# General Agenda

* Agenda Items from comments left on this Github issue
* Additional in-meeting comments / discussion
* [If time allows] Recap of [WebAssembly CG meeting](https://github.com/WebAssembly/meetings) if anyone attended

Feel free to comment on this issue if you have any agenda items you would like to bring up. 
Meeting Notes will be placed on this issue for those who cannot make the meeting.

# Meeting Information

We will use the AssemblyScript discord for meetings: https://discord.gg/vSBbJ5T . We will meet in the "meeting" voice channel :+1: 

Anyone who is contributing to the AssemblyScript project, building something with AssemblyScript, interested in the Assembly project or WebAssembly in general, is welcome to join! 😄

Notes will be taken by the host, and posted after the meeting. Notes are free to be edited through comments on the meeting notes at a later time.

**Comment by torch2424** [comment]
# Meeting Notes

* @battlelinegames : Working on the WebGL Bindings
   * Got unblocked, but just tying up the loose ends :)
* @battlelinegames : Bytecode Alliance?
   * Haven’t been in meetings or anything, they’ve been having a lot of internal work being done, but still very interested
* Rick & @saulecabrera : Language Server? Which extension should we be using?
   * @saulecabrera : We currently support both both ts and as, but I can change it at anytime
   * @battlelinegames : I just want my Assembly script to be handled, not Typescript
   * @saulecabrera : Yeah, and right now you can’t mix and match extensions
   * @battlelinegames : Code completion is a bit awkward using a `.ts` in vscode
   * @torch2424 : https://github.com/AssemblyScript/assemblyscript/issues/1003 
      * https://github.com/github/linguist/blob/master/lib/linguist/languages.yml 
   * I think we might be good to go to actually develop a strategy on this for a final extension. :) Now I’m kinda leaning towards a comma separated list of extensions, so we don’t break the current ecosystem.
* @torch2424 : Writing Article for Porting JavaScript to AssemblyScript
   * Going to write an “execution time” app, in both node and AS
   * And port the “pretty-ms” JS NPM module, to AS to show how it can be ported.
* @torch2424 : Working on a URL Parsing library
   * Will replicate the functionality of: https://developer.mozilla.org/en-US/docs/Web/API/URL in Assembly script
* @torch2424 : Also, Organizing Wasm Summit 2021
* @saulecabrera : Internationalization in Assembly script?
   * @willemneal & @torch2424 : Not that we know of?
   * @saulecabrera : We need a linear at runtime solution for grabbing text for localization in Assembly script (Such as a gettext())
   * @willemneal : If you want to use loaders, check out my visitor-as
      * https://github.com/willemneal/visitor-as 
   * @torch2424 : You can generate Assembly script source code, as a Map<string,string> and import that
   * @saulecabrera : We use .po so, we’d need a solution for that
* @willemneal : Asbuild now in the AS org
   * Also planning on adding more features :) In particular `asinit`
   * Want to improve transforms detection 
      * @saulecabrera : That would be super useful for us :)
* @willemneal : Snapshotting tests
   * Something I’m working on :)

**Comment by torch2424** [comment]
Closing in favor of: https://github.com/AssemblyScript/working-group/issues/51

Feel free to continue discussions from this meeting here :smile:
